component-classes/index.js

```diff
@@ -20,15 +20,17 @@ export const ribbon = {
 };
 
 export const badge = {
-  base: 'py-4 px-8 border-0 rounded-4 text-xs inline-flex',
-  neutral: 'i-bg-$color-badge-neutral-background i-text-$color-badge-neutral-text',
-  info: 'i-bg-$color-badge-info-background i-text-$color-badge-info-text',
-  positive: 'i-bg-$color-badge-positive-background i-text-$color-badge-positive-text',
-  warning: 'i-bg-$color-badge-warning-background i-text-$color-badge-warning-text',
-  negative: 'i-bg-$color-badge-negative-background i-text-$color-badge-negative-text',
-  disabled: 'i-bg-$color-badge-disabled-background i-text-$color-badge-disabled-text',
-  price: 'i-bg-$color-badge-price-background i-text-$color-badge-price-text',
-  notification: 'i-bg-$color-badge-notification-background i-text-$color-badge-notification-text',
+  base: 'py-4 px-8 border-0 rounded-4 text-xs inline-flex s-text',
+  neutral: 'bg-[--w-color-badge-neutral-background]',
+  info: 'bg-[--w-color-badge-info-background]',
+  positive: 'bg-[--w-color-badge-positive-background]',
+  warning: 'bg-[--w-color-badge-warning-background]',
+  negative: 'bg-[--w-color-badge-negative-background]',
+  disabled: 's-bg-disabled',
+  price: 'bg-[--w-color-badge-price-background] s-text-inverted',
+  sponsored: 'bg-[--w-color-badge-sponsored-background]',
+  // Notification badge is deprecated: Do not use!
+  notification: 'bg-[--w-color-badge-notification-background]',
```

Comment:
Shouldn't it still include `s-color-text-notification`?

Reply:
Yes, true 👍 🙏